src/app/about/page.js

```diff
@@ -1,10 +1,23 @@
-import Image from "next/image";
-import Fall231 from "@/public/images/fall/fall23_1.webp";
+import Fall231 from "@/public/images/fall/fall23_1.webp"; // thisp picture was laready here when i opened file idk if this is what im supposed to put
+import { ABOUT } from "@/data/aboutPage";
+import Title from "@/components/Title";
+import Picture from "@/components/Picture";
 
 const Page = () => {
   return (
-    <div className="bg-biscuits-pink bg-blobs1 md:bg-cover bg-contain bg-repeat-y">
-      <Image src={Fall231} width={500} height={500} />
+    <div className="bg-biscuits-green-100 grid grid-cols-1 gap-y-10 p-4 md:p-10">
+      <Title text="About Us" color="red" />
+      <div className="flex justify-center">
+        <Picture x={10} y={10} image={Fall231} alt="Fall 23" />
+      </div>
+      {ABOUT.map((section, index) => (
+        <div key={index} className="mb-8 text-center">
+          <h2 className="italic text-biscuits-green-200 font-shrikhand text-4xl mb-4">
+            {section.subtitle}
+          </h2>
+          <p className="text-base font-inika">{section.paragraph}</p>
```

Comment:
change to div

Reply:
text size should be 2xl

Reply:
also use width to squish the paragraphs, which will make the paragraphs less long. maybe w-2/3 will work.

![image](https://github.com/user-attachments/assets/526afe16-7504-4e54-8859-f128ad1da828)